From ea908a95a3472db929a667b959bd0f09e676bba5 Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Mon, 20 Mar 2023 21:02:23 +0200
Subject: [PATCH 2/2] Drop Designer plugin as it fails to compile

---
 src/plugins/plugins.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/plugins.pro b/src/plugins/plugins.pro
index 50181aa19..66ba995e1 100644
--- a/src/plugins/plugins.pro
+++ b/src/plugins/plugins.pro
@@ -1,3 +1,3 @@
 TEMPLATE = subdirs
-qtHaveModule(webenginewidgets): qtHaveModule(designer): SUBDIRS += qwebengineview
+# qtHaveModule(webenginewidgets): qtHaveModule(designer): SUBDIRS += qwebengineview
 qtHaveModule(pdf): qtConfig(imageformatplugin): SUBDIRS += imageformats
-- 
2.39.2

